orchestrator/api/cloud_documents.py

```
@@ -417,7 +417,8 @@ async def trigger_sync(
     except HTTPException:
         raise
     except ValueError as e:
-        raise HTTPException(status_code=400, detail=str(e))
+        logger.warning(f"Cloud document sync validation error: {e}")
+        raise HTTPException(status_code=400, detail="Invalid sync request")
```

Ruff (0.15.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)